srcpkgs/hunspell-el_GR/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'hunspell-el_GR'
+pkgname=hunspell-el_GR
+version=0.4.0
+revision=1
+_version=0.4.0-87
```

Comment:
I would rather do something similar to libcli, so only `version` would be needed.

Reply:
Okay, I will amend it accordingly.